conf/application.conf

```diff
@@ -56,7 +56,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
 # ~~~~~
 # The secret key is used to secure cryptographics functions.
 # If you deploy your application to several instances be sure to use the same key!
-application.secret = "WPlVxIW4BicDrNKU16zne2PBoexBdh1LXuKdfnZOoVk02DoyECB26jqYPoFW5LcB"
+play.crypto.secret = "WPlVxIW4BicDrNKU16zne2PBoexBdh1LXuKdfnZOoVk02DoyECB26jqYPoFW5LcB"
```

Comment:
Hi, I saw this commit pop up on twitter ( https://twitter.com/havemysecrets/status/1166721635573014528 ). What does this key give access to? Is it a dummy value used for testing?